src/index.js

```diff
@@ -0,0 +1,111 @@
+import express from 'express';
+import { v4 as uuidv4 } from 'uuid';
+import _ from 'lodash';
+
+const PORT = Number(process.env.PORT) || 3000;
+const app = express();
+app.use(express.json());
```

Comment:
also add unhandled exception handler. For now, if exception happens, your API returns HTML

Reply:
Added error handler.